### ISSUE TITLE
Unable to query Shabads with Translations/Transliterations on Banis query

### ISSUE THREAD

**Issue by Sarabveer** [ShabadOS/database#206]
```javascript
Banis.query()
    .where( 'banis.id', baniId )
    .eager( 'lines.shabad.[section, source]' )
    .withTranslations( translationSources )
    .withTransliterations( [ 1, 4 ] )
    .orderBy( [ 'line_group', 'lines.order_id' ] )
    .then( ( [ bani ] ) => bani )
```

This query errors out with `ambiguous column name: order_id`. There is a conflict because if `.withTranslations()` and ` .withTransliterations()` are removed, the query runs fine.

**Comment by Harjot1Singh** [comment]
Because `minimize` is set to `true` in `eagerOptions` when `withTranslation` or `withTransliteration` is applied...

```
export const getBaniLines = baniId => Banis
  .query()
  .joinEager( 'lines.shabad' )
  .orderBy( [ 'line_group', 'l.order_id' ] )
  .where( 'banis.id', baniId )
  .withTranslations()
  .withTransliterations()
  .eagerOptions( { minimize: false, aliases: { lines: 'l' } } )
  .then( ( [ bani ] ) => bani )
```
fixes this, but a more robust solution/less manual needs to be implemented

**Comment by Harjot1Singh** [comment]
Fixed in 7b2d7b580fbdc7ba0c24c8b226be235afecda2c4
